recipes-devtools/pcie-debug/pcie-debug_git.bb

```diff
@@ -0,0 +1,21 @@
+# Copyright (C) 2019 Viktor Sjölind <vsjolind@luxoft.com>
+# Released under the MIT license (see COPYING.MIT for the terms)
+
```

Comment:
btw, this recipe should probably go to meta-bistro

Reply:
Yes you are right